Gifski/Info.plist

```diff
@@ -23,6 +23,26 @@
 			</array>
 		</dict>
 	</array>
+	<key>NSServices</key>
+	<array>
+		<dict>
+			<key>NSMessage</key>
+			<string>convertToGif</string>
+			<key>NSPortName</key>
+			<string>Gifski</string>
+			<key>NSMenuItem</key>
+			<dict>
+				<key>default</key>
+				<string>Gifski: Convert to GIF</string>
+			</dict>
+			<key>NSRequiredContext</key>
+			<dict/>
+			<key>NSSendFileTypes</key>
+			<array>
+				<string>public.movie</string>
```

Comment:
We don't support all types of movies, so it should be the same as https://github.com/sindresorhus/gifski-app/blob/62877fb387299030d780cbee5df13ebb84eebf36/Gifski/Info.plist#L16-L18

Reply:
Good catch 👍